imagersite/imager_images/tests/test_photo.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import unicode_literals
+from django.test import TestCase, override_settings
+from imager_images.models import Photo, Album
+from imager_images.tests.factories import PhotoFactory, UserFactory
+import tempfile
+import datetime
+
+
+# Create your tests here.
+
+
+class PhotoTestCase(TestCase):
+    '''This is the test case for the Photo Model'''
+
+    import pdb; pdb.set_trace()
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    @override_settings(MEDIA_ROOT=tempfile.mkdtemp())
+    def test_upload_photo(self):
+        new_photo = PhotoFactory()
+        photo_dir = 'photo_files'
+        user = UserFactory.build()
```

Comment:
Why not put your user and photo creation in the `setUp` function? That way for every test you run you'd always have the same access to the same user and the same photo.